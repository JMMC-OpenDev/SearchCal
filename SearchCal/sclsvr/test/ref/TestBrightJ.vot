version https://git-lfs.github.com/spec/v1
oid sha256:cc348f7d09781dac2566ee7d91b73aea8b84c24d0e7a02c86fc9ad39767fc502
size 2223430
