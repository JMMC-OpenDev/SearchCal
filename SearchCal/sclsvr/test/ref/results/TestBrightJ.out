version https://git-lfs.github.com/spec/v1
oid sha256:7bf9b6f3788bfd4181ec26a47e23c3a4940d7f9d9ff4712402ef0d86e207849f
size 1447154
